src/pages/menuUpload/orderDetails/OrderLineDetails.tsx

```diff
@@ -0,0 +1,280 @@
+import React, {
+  useCallback, useEffect, useRef, useState,
+} from 'react';
+
+import { RiDeleteBin7Line } from 'react-icons/ri';
+import { useIntl } from 'react-intl';
+import { useSelector } from 'react-redux';
+
+import BatchNumberPicker from 'components/pickers/BatchNumberPicker';
+import ItemPickerOutsideForm from 'components/pickers/ItemPickerOutsideForm';
+import EditableTable from 'components/shared/table/EditableTable';
+import Input from 'shared/inputs/input';
+import { RootState } from 'stores/reducers/rootReducer';
+import { orderLineDetailsUrl } from 'stores/sagas/orderLineDetailsSaga';
+import refreshAccessToken from 'stores/sagas/utils';
+import { getRequest } from 'utils/apiClient';
+import { DEFAULT_OFFSET, EMPTY_SEARCHER, FIRST_PAGE } from 'utils/consts';
+
+type Props = {
+  mode: 'Edit' | 'Add'
+  orderDetailsId: string | null
+  onLineItemsChange: (lineItems: any) => void
+};
+
+function OrderLineDetails(
```

Comment:
I wonder whether should we move this part into 'Form'. Because this is a EditableTable (part of Form), not a whole page component. It shouldn't affect on current solution. But if you have a good explanation we can keep like this for this moment because of our timeline.

Reply:
I can move this file :)

Reply:
perfect! thank you!